vpr/src/draw/draw.cpp

```diff
@@ -998,16 +998,16 @@ static void drawplace(ezgl::renderer* g) {
                 ezgl::color block_color;
                 t_logical_block_type_ptr logical_block_type = nullptr;
 #    ifdef VTR_ENABLE_DEBUG_LOGGING
-                if (f_placer_debug) {
-                    t_pl_loc curr_loc;
-                    curr_loc.x = i;
-                    curr_loc.y = j;
-                    auto it = std::find_if(draw_state->colored_blocks.begin(), draw_state->colored_blocks.end(), [&curr_loc](const std::pair<t_pl_loc, ezgl::color>& a) { return (a.first.x == curr_loc.x && a.first.y == curr_loc.y); });
-                    if (it != draw_state->colored_blocks.end()) {
+                t_pl_loc curr_loc;
+                curr_loc.x = i;
+                curr_loc.y = j;
+                auto it = std::find_if(draw_state->colored_blocks.begin(), draw_state->colored_blocks.end(), [&curr_loc](const std::pair<t_pl_loc, ezgl::color>& a) { return (a.first.x == curr_loc.x && a.first.y == curr_loc.y); });
+                if (placer_debug_enabled() && it != draw_state->colored_blocks.end()) {
```

Comment:
Is placer_debug_enabled() the right name?  Does this really mean we are in a breakpoint?  If so worth renaming.